modules/system/console/MixCompile.php

```diff
@@ -101,7 +102,9 @@ public function handle(): int
                 continue;
             }
 
-            $this->info(sprintf('Mixing package "%s"', $name));
+            if (!$this->option('no-progress')) {
+                $this->info(sprintf('Mixing package "%s"', $name));
+            }
```

Comment:
Perhaps this should be hidden only when `silent` mode is defined, but should still show when `no-progress` is defined. Silent implies no (or very limited) output, but no progress just means we hide any progress bars from showing.

```suggestion
            if (!$this->option('silent')) {
                $this->info(sprintf('Mixing package "%s"', $name));
            }
```